alchemist-full/build.gradle.kts

```diff
@@ -145,6 +146,42 @@ tasks.register<Exec>("testJpackageOutput") {
     finalizedBy(deleteJpackageOutput)
 }
 
+val generatePKGBUILD by tasks.registering(Exec::class) {
+    group = "Publishing"
+    description = "Generates a valid PKGBUILD by replacing values in the template file"
+    val inputFile = file("${project.projectDir}/PKGBUILD.template")
+    val outputDir = file(rootProject.layout.buildDirectory.map { it.dir("package") }.get().asFile.path)
+    val tokenToReplace = "{%}"
+    var replacementValues: List<String>
+    val testSourceParam = System.getProperty("generatePKGBUILD.testSource", "false")
+    val interceptOutput = ByteArrayOutputStream()
+    standardOutput = interceptOutput
+    doFirst {
+        val version = rootProject.version.toString().substringBefore('-')
+        commandLine("md5sum", "-b", "${rootProject.layout.buildDirectory.get().asFile.resolve("package").path}/${rootProject.name}-$version-1.x86_64.rpm")
```

Comment:
why do we need `-1` at the end?

Reply:
Jpackage outputs packages following that syntax. I suppose it's the release number.

Reply:
There must be a reason, can you investigate? If there is no reason for it to have -1, I'd like to remove it

Reply:
It is a naming convention for .rpm packages, .deb packages follow something similar too

> This is tradition and not a restriction, however. Like the version number, the only restriction is that dashes are not allowed. [source](http://ftp.rpm.org/max-rpm/ch-rpm-file-format.html)

I found the jpackage setting, but it seems impossible to disable. 
We can modify it to whatever string we would like